src/next/auth-wrapper.ts

```diff
@@ -17,13 +18,17 @@ export function withMcpAuth(
   return async (req: Request) => {
     const origin = new URL(req.url).origin;
 
-    const authInfo = await verifyToken(req);
+    const authHeader = req.headers.get("Authorization");
+    const [_, bearerToken] = authHeader?.split(" ") || [];
```

Comment:
Should we check for "Bearer"? Or alternatively we could pass `token: {type: string; value: string}`?